pandas/core/indexes/period.py

```diff
@@ -133,29 +133,43 @@ class PeriodIndex(DatelikeOps, DatetimeIndexOpsMixin, Int64Index):
     Parameters
     ----------
     data : array-like (1-dimensional), optional
-        Optional period-like data to construct index with
+        Optional period-like data to construct index with.
     copy : bool
-        Make a copy of input ndarray
+        Make a copy of input ndarray.
     freq : string or period object, optional
-        One of pandas period strings or corresponding objects
+        One of pandas period strings or corresponding objects.
     start : starting value, period-like, optional
         If data is None, used as the start point in generating regular
         period data.
     periods : int, optional, > 0
         Number of periods to generate, if generating index. Takes precedence
-        over end argument
+        over end argument.
     end : end value, period-like, optional
         If periods is none, generated index will extend to first conforming
-        period on or just past end argument
+        period on or just past end argument.
     year : int, array, or Series, default None
+        If not None requires a freq to be specified and either
+        month or quarter to be specified.
     month : int, array, or Series, default None
+        If not None requires a freq and year to be specified.
     quarter : int, array, or Series, default None
+        If quarter is not None the freq is required to be
+        quarterly Q or Q-DEC. it will be set to Q-DEC if not specified.
+        day, hour, minute, second are not used.
     day : int, array, or Series, default None
+        If not None requires a freq, month, and year to be specified.
+        day defaults to 1 if year and month specified.
     hour : int, array, or Series, default None
+        If not None requires a freq, month, and year to be specified.
```

Comment:
these also require a day from hour on down

Reply:
No a day is not required from hour since day will take a default value of 1. This is why I specified that a different default value is taken if year and month are specified.